src/components/PostList/PostList.jsx

```diff
@@ -1 +1,10 @@
-export const PostList = () => <>Put the list here</>;
+import { PostInfo } from '../PostInfo';
+
+export const PostsList = ({ posts }) => (
+  <>
+    {posts.map(post => (
+      <PostInfo key={post.id} post={post} />
+    ))}
+    ;
```

Comment:
The semicolon here is unnecessary and can be removed to avoid confusion. It is not needed in JSX syntax.